### PR TITLE
[WIP]: Fixes #1 Deleted Base_StocksDashboard class and added variables to init

### LINKED ISSUE #1: Refactor code
Implement refactoring of the existing code and reorganize/create new classes.

### PR DESCRIPTION
**Pull request by mabelvj**
- Removed Base_StocksDashboard.
- Modified StocksDashboard `__init__` to add the allowed variables.

### CODE REVIEW

File: StocksDashboard.py [hunk 1]
```diff
@@ -15,7 +14,7 @@
 HEIGHT = 648
 
 
-__all__ = ['Base_StocksDashboard', 'StocksDashboard', 'datetime', 'get_colors']
+__all__ = ['StocksDashboard', 'datetime', 'get_colors']
 
 
 def datetime(x):
```
Comment (EliosMolina) [inline StocksDashboard.py]: This name can be confused with standard datetime object (https://docs.python.org/2/library/datetime.html)

File: StocksDashboard.py [hunk 2]
```diff
@@ -46,9 +45,21 @@ class Base_StocksDashboard():
...
 
-    def __init__(self, **kwargs):
-        for k, v in kwargs.items():
-            setattr(self, k, v)
+    width = WIDTH
+    height = HEIGHT
+    ncols = 1
+    names = None
+
+    # code = "source.set('selected', cb_data['index']);"
```
Comment (EliosMolina) [inline StocksDashboard.py]: These comments should be out

File: StocksDashboard.py [hunk 3]
```diff
@@ -46,9 +45,21 @@ class Base_StocksDashboard():
...
+    # callback = CustomJS(args={'source': source}, code=code)
+
+    def __init__(self, width=None, height=None, ncols=None):
```
Comment (EliosMolina) [inline StocksDashboard.py]: You could do: 
```
def __init__(self, width=WIDTH, height=HEIGHT, ncols=1):
    self.width = width
    self.height = height
    self.ncols = ncols
```
Then you can remove the declaration of the members of the class (line 48...).

File: StocksDashboard.py [hunk 4]
```diff
@@ -46,9 +45,21 @@ class Base_StocksDashboard():
...
+        if width:
+            self.width = width
+        if height:
+            self.height = height
+        if ncols:
+            self.ncols = ncols
 
     @staticmethod
     def create_hover(tooltips=[('date', '$x{%F}'), ('value', '@y{0.000}')],
```
[the remark this answers is not in the document]
Comment (mabelvj) [reply StocksDashboard.py]: There is plenty of formatting that can be done, and also, we can reference the lines on which we allow the tooltip to show. Leave you a link [`class HoverTool(**kwargs)`](https://bokeh.pydata.org/en/latest/docs/reference/models/tools.html#bokeh.models.tools.HoverTool). Maybe we should reference this in the Documentation.

[the remark this answers is not in the document]
Comment (EliosMolina) [reply StocksDashboard.py]: Ok, but maybe this way?
```
    @staticmethod
    def create_hover(tooltips=[('date', '$x{%F}'), ('value', '@y{0.000}')],
                     formatters={'$x': 'datetime'}, mode='vline', **kwargs):
        hover = HoverTool(**kwargs)
        hover.tooltips = tooltips
        hover.formatters = formatters
        hover.mode = mode
        return hover
```

File: StocksDashboard.py (hunk 1 above)
Comment (mabelvj) [reply StocksDashboard.py]: ok, I took it from this example in bokeh [stocks.py](https://bokeh.pydata.org/en/latest/docs/gallery/stocks.html). I'll add it as a static method then.

File: StocksDashboard.py (hunk 3 above)
Comment (mabelvj) [reply StocksDashboard.py]: ok, but then we should check these values somehow.

**Pull request merged**